config-service-impl/src/main/java/org/hypertrace/config/service/store/DocumentConfigStore.java

```diff
@@ -166,6 +175,23 @@ public ContextSpecificConfig getConfig(ConfigResourceContext configResourceConte
         .orElseGet(() -> emptyConfig(configResourceContext.getContext()));
   }
 
+  @Override
+  public Map<ConfigResourceContext, ContextSpecificConfig> getAllContextConfigs(
+      Set<ConfigResourceContext> configResourceContexts) throws IOException {
+    Map<ConfigResourceContext, Optional<ConfigDocument>> configDocs =
+        getLatestVersionConfigDocs(configResourceContexts);
+
+    return configDocs.entrySet().stream()
```

Comment:
Consider Maps.transformValues

Reply:
problem would be null values. i need to filter it out later

Reply:
If the response has a different keyset than the source, then yes you'd probably use a collector. In the current code at least, that's not the case - you're strictly transforming the value.